File: NapakalakiRuby/lib/prize.rb
#encoding: utf-8
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module NapakalakiGame

class Prize
  def initialize(t, l)
    @treasures = t
    @levels = l
  end
  
  def getTreasures
    return @treasures
  end
  
  def getLevels
    return @levels
  end
  
  def to_s
    cadena = "Treasures = #{@treasures} / Levels = #{@levels}"
    return cadena 
  end
end 

end

File: NapakalakiJava/src/NapakalakiGame/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author danibolanos
 */

public class Prize {
    private int treasures;
    private int levels;
    
    public Prize(int t, int l){
        treasures = t;
        levels = l;
    }
    
    public int getTreasures(){
        return treasures;
    }
            
    public int getLevels(){
        return levels;
    }
    
    public String toString(){
        return "Treasures = " + Integer.toString(treasures) + " / Levels = " 
        + Integer.toString(levels);
    }
}
